recipes/hexl/all/conanfile.py

```diff
@@ -0,0 +1,161 @@
+from conans import CMake, tools
+from conan import ConanFile
+from conans.errors import ConanInvalidConfiguration
```

Comment:
```suggestion
from cona import ConanFile
from conan.errors import ConanInvalidConfiguration
from conan.tools.microsoft import is_msvc
```
get prepared for Conan 2.0